app/src/main/res/layout/problem_map_info_window.xml

```diff
@@ -67,6 +67,9 @@
         android:id="@+id/problem_map_info_window_content_thumb"
         android:layout_width="100dp"
         android:layout_height="match_parent"
+        android:layout_marginBottom="2dp"
+        android:layout_marginRight="2dp"
+        android:layout_marginTop="2dp"
```

Comment:
Yes, it is. Design has report content on the left and photo on the right - so I didn't want double paddings/margins for both items. That's why the photo does not have anything on the left.